pyproject.toml

```diff
@@ -187,6 +187,7 @@ allow_untyped_defs = false
 [tool.codespell]
 ignore-words-list = "coo,nd"
 skip = "docs/source/changelog.rst"
+exclude-file = "docs/source/index.rst"
```

Comment:
This is not quite what I was going for... spellcheck doesn't like `medias` on line 21, but when I tried `exclude-file = "docs/source/index.rst:21"` codespell couldn't find the file.

Maybe there's a better way to ignore all .. raw:: html content with a regex or something?